ocs_ci/framework/main.py

```diff
@@ -154,6 +154,11 @@ def process_ocsci_conf(arguments):
 
     args, unknown = parser.parse_known_args(args=arguments)
     load_config(args.ocsci_conf)
+    if unknown:
+        for each_arg in unknown:
+            if each_arg.startswith("--cluster-path="):
```

Comment:
what about scenario of --cluster-path /my/cluster path?

Then you will not split with = but space in line bellow, should we tackle also this scenario?

Reply:
done in latest commit